.travis.yml

```diff
@@ -37,7 +37,11 @@ _android: &_android
 
 matrix:
   include:
-    # one local test, without saucelabs
+    # additional tests via package.json
+    - env: ADDITIONAL_TESTS=./spec/testable-plugin/tests/foo
```

Comment:
Variable name is confusing, I would expect a command to execute not the directory where the tests are located.

Reply:
`ADDITIONAL_TESTS_DIR`?

Reply:
Sounds good!